src/documents/InclusiveMakingAndHacking.html.md.handlebars

```diff
@@ -0,0 +1,205 @@
+
+Secondly, designs intended to serve the needs or goals of individuals on the margin can turn out to be of much broader use. This is sometimes referred to as the *curb-cut effect* after the sidewalk ramps that were originally intended for wheelchair users, but have ended up being of significant benefit to many other users of public spaces.
+
+### Considering the Full Range of Human Diversity
+
+Because we define Inclusive Design as design that considers the full range of human diversity with respect to ability, language, culture, gender, age and other forms of human difference, our vision for inclusive hackathons and hacking must encompass more than the (vital!) practice of accomodation for people with disabilities through the design of the physical space, the preparation of alternate formats for information, and services such as sign language interpretation.
+
+Several of the resources at the bottom of this entry contain advice for designing hackathon spaces and structuring events to improve accessibility for people with disabilities. We would add two pieces of specific advice from Inclusive Design practice:
+
+* Use a comprehensive approach like that embodied in the <a href="https://guide.inclusivedesign.ca/activities/InclusiveDesignMapping.html" target="_blank" class="link-external">Inclusive Design Mapping</a> to reconsider and stretch the design of the hackathon or hacking exercise towards greater inclusion.
+
+* Plan, execute and review in the open; give people opportunity to provide feedback before and after the event or activity. A living document supported by technology such as a wiki can help to invite necessary commentary and revision.
+
+#### On STEM Diversity Issues
+
+Because hacking and making are strongly associated with STEM (Science, Technology, Engineering and Math) disciplines, it's not possible to discuss this subject without reference to the widespread work that has gone on to investigate and address the issue of demographic diversity in STEM both academically and <a href="https://www.wired.com/brandlab/2015/05/5-numbers-explain-stem-diversity-matters-us/" target="_blank" class="link-external">in the wider world</a>.
+
+The STEM acronym itself has come under some criticism for excluding art and design, leading to initiatives such as the <a href="http://stemtosteam.org/" target="_blank" class="link-external">STEM to STEAM Initiative</a>.
+
+We encourage those considering hacking and making in learning contexts to be aware of larger issues of diversity in STEM subjects, workplaces and communities, as these issues can have relevance and knock-on effects to the diversity and inclusion of hackathons and hacking.
+
+## Tools
+
+The sections below include examples of tools (software, hardware and design-related) that can be useful in inclusive design hacking. Some are early-stage research projects, while others are mature and in widespread usage in and outside the classroom.
```

Comment:
How about: "The sections below include examples of software, hardware, and design tools that may be useful for inclusive design hacking."?